library/src/scala/tasty/reflect/TypeOrBoundsOps.scala

```diff
@@ -84,8 +85,8 @@ trait TypeOrBoundsOps extends Core {
     }
 
     object TermRef {
-      def unapply(typeOrBounds: TypeOrBounds) given (ctx: Context): Option[(Symbol, TypeOrBounds /* Type | NoPrefix */)] =
-        internal.matchTermRef_unapply(typeOrBounds)
+      def unapply(typeOrBounds: TypeOrBounds) given (ctx: Context): Option[(TypeOrBounds /* Type | NoPrefix */, String)] =
+        internal.matchTermRef(typeOrBounds).map(x => (x.qualifier, x.name))
```

Comment:
The symbol is not part of the type structure. It is also more efficient to get the symbol directly with `termSymbol` rather than boxing it with the qualifier.